lighthouse-core/test/fixtures/invalid-audits/missing-description.js

```diff
@@ -11,8 +11,9 @@ const LighthouseAudit = require('../../../audits/audit');
 class MissingRequiredArtifacts extends LighthouseAudit {
   static get meta() {
     return {
-      name: 'missing-description',
-      helpText: 'This is missing required description (and failure description)',
+      id: 'missing-description',
+      title: 'Missing required artifacts',
```

Comment:
description

Reply:
err.. if its the same test then we'd leave out `title` and `failureTitle` but provide a description.

Reply:
Ha, I meant that the title said it was missing artifacts but good point

Reply:
we already have `missing-title` for that, this one is really the help text check, git diff is confused